tao_compiler/mlir/disc/transforms/disc_unhandled_atomic_rmw_converter.cc

```diff
@@ -44,9 +44,12 @@ struct UnhandledAtomicRMWConverter
 
   LogicalResult matchAndRewrite(memref::AtomicRMWOp op,
                                 PatternRewriter& rewriter) const override {
+    Type elemTy = op.getType();
```

Comment:
better to add a i8 sum e2e test like here: https://github.com/alibaba/BladeDISC/blob/main/tao_compiler/mlir/disc/tests/tensorflow_ops/sum.cc#L25

Reply:
done